<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLCDNumber" name="lcdNumber"/>
   </item>
   <item row="1" column="0">
    <widget class="QLCDNumber" name="lcdNumber_2"/>
   </item>
   <item row="2" column="0">
    <widget class="QLCDNumber" name="lcdNumber_3"/>
   </item>
   <item row="3" column="0">
    <widget class="QLCDNumber" name="lcdNumber_4"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
